src/app/home/navigation.component.ts

```diff
@@ -34,6 +54,16 @@ export class NavigationComponent implements OnInit {
     { link: '', name: 'Feedback' },
   ];
 
+  ngOnInit() {
+    Object.assign(this, this.userService.get());
+    this.languages = (<any>languages);
+  }
+
+  switchLanguage(lang) {

```

Comment:
later once we have AOT it should change the url of application
And selected language needs to be set based on the url of application